packages/react-ui-ag/src/Listings/MobileMapListing.js

```diff
@@ -15,7 +15,14 @@ const buttonPropTypes = PropTypes.shape({
   valueLocation: PropTypes.string,
 })
 
-@themed(/^MobileMapListing/)
+const REACT_LAZYLOAD_PROP_DEFAULTS = {
+  offset: [250, -100],
+  resize: true,
+  width: 280,
+  height: 120,
+}
+
+@themed(/^MobileMapListing/, { pure: true })
```

Comment:
This isn't a pure component so you should remove `{ pure: true }`

Reply:
Done. #mergingconflictsfrommasterbranchishard :(